models/op/governance/optimism/op_governance_optimism_delegators_incremental.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        tags=['dunesql']
+        , schema = 'op_governance_optimism'
+        , alias = alias('delegators_incremental')
```

Comment:
what's the intention of `_incremental` in the table name (and file name)? is that needed?